tests/fsharp/single-test.fs

```diff
@@ -71,54 +69,49 @@ let generateProps testCompilerVersion configuration =
   <Import Project=""$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(PROJECTDIRECTORY)'))"" />
 </Project>"
     template
-    |> replaceTokens "$(TESTCONFIGURATION)" configuration
-    |> replaceTokens "$(PROJECTDIRECTORY)" (Path.GetFullPath(__SOURCE_DIRECTORY__))
-    |> replaceTokens "$(TESTCOMPILERVERSION)" testCompilerVersion
+      .Replace("$(TESTCONFIGURATION)", configuration)
```

Comment:
@KevinRansom If you're wondering about this bit of bike shedding - basically it's fine to use fluent notation `Replace` (and preferable over a pipelined function that doesn't specifically do anything except a Replace)